linalg/blockoperator.hpp

```diff
@@ -153,11 +153,11 @@ class BlockDiagonalPreconditioner : public Solver
    //! Add a square block op in the block-entry (iblock, iblock).
    /**
     * iblock: The block will be inserted in location (iblock, iblock).
-    * op: the Operator to be inserted.
+    * opt: the Operator to be inserted.
     */
-   void SetDiagonalBlock(int iblock, Operator *op);
+   void SetDiagonalBlock(int iblock, Operator *opt);
```

Comment:
It may make more sense to rename the class variable `ops`

Reply:
Good point